sdk/inc/azure/storage/az_storage_blobs.h

```diff
@@ -68,29 +66,30 @@ typedef struct
 /**
  * @brief Initialize a client with default options.
  *
- * @param client The blob client instance to initialize.
- * @param endpoint A url to a blob storage account.
- * @param credential The object used for authentication.
- *         #AZ_CREDENTIAL_ANONYMOUS should be used for SAS.
- * @param options A reference to an #az_storage_blobs_blob_client_options structure which defines
- * custom behavior of the client.
+ * @param[out] out_client The blob client instance to initialize.
+ * @param[in] endpoint A URL to a blob storage account.
+ * @param credential The object used for authentication. #AZ_CREDENTIAL_ANONYMOUS should be used for
```

Comment:
[in]

Reply:
The reason I didn't mark it is that logically it does not mutate, but factually it does. But since all the mutations are in the `_internal`, these can be marked as `[in]` I think.